libyul/backends/evm/EVMDialect.cpp

```diff
@@ -41,15 +44,26 @@ using namespace solidity::util;
 namespace
 {
 
-std::pair<YulName, BuiltinFunctionForEVM> createEVMFunction(
+size_t toContinuousVerbatimIndex(size_t _arguments, size_t _returnVariables)
+{
+	return _arguments + _returnVariables * Dialect::verbatimMaxInputSlots;
+}
+
+std::tuple<size_t, size_t> verbatimIndexToArgsAndRets(size_t _index)
+{
+	auto const numRets = static_cast<size_t>(std::floor(_index / EVMDialect::verbatimMaxInputSlots));
```

Comment:
You don't need `floor()` here. You're doing integer division, which truncates and stays within `size_t`. All `floor()` does here is convert the argument to a floating point number, which you then have to convert back :)

```suggestion
	size_t const numRets = _index / EVMDialect::verbatimMaxInputSlots;
```

Reply:
Good catch - I didn't think about it being unsigned integers. For signed ones you always have to be careful around zero with that truncation. Then the floor would be required, I think. In any case this even allows it to be constexpr. Fixed!